### DISCUSSION TITLE
firewall and mirror port

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#4503] [General]
Hi pals!
I have this simple qst. I'm deploying SO standalone mode to make a POC. My arquitecture is VMware workstation, SO and a win10 machine to generate events. I really don't want to install a firewall like pfsense o similar. Is it posible to configure security onion to work as packet forwarding and set in my win10 VM, security onion management ip as DG?
Thanks in advanced
Mariano

**Comment by weslambert** [comment]
You should be able set up a virtual switch/dedicated network in VMWare Workstation and connect any necessary VMs to that for mirroring purposes.
